sources/packages/backend/apps/api/src/constants/error-code.constants.ts

```diff
@@ -239,6 +239,7 @@ export const APPLICATION_RESTRICTION_BYPASS_NOT_FOUND =
  */
 export const APPLICATION_RESTRICTION_BYPASS_IS_NOT_ACTIVE =
   "APPLICATION_RESTRICTION_BYPASS_IS_NOT_ACTIVE";
+
```

Comment:
Minor, not a blocker, no real changes in this file that would require its inclusion in the PR.

Reply:
This is fix from comment mentioned in previous PR for same story.

https://github.com/bcgov/SIMS/pull/4011#discussion_r1862674114